source/frontend/src/features/mapSideBar/research/ResearchContainer.tsx

```diff
@@ -77,17 +92,42 @@ export const ResearchContainer: React.FunctionComponent<
     retrieved?.fileProperties?.forEach(async fp => {
       fp.property = researchProperties?.find(ap => fp.id === ap.id)?.property;
     });
-    setResearchFile(retrieved);
     setFile({ ...retrieved, fileType: FileTypes.Research });
-    setStaleFile(false);
-  }, [getResearchFile, getResearchFileProperties, props.researchFileId, setFile, setStaleFile]);
+  }, [getResearchFile, getResearchFileProperties, props.researchFileId, setFile]);
+
+  const fetchLastUpdatedBy = React.useCallback(async () => {
+    var retrieved = await getLastUpdatedBy(props.researchFileId);
+    if (retrieved !== undefined) {
+      setLastUpdatedBy(retrieved);
```

Comment:
strictly speaking if you aren't using the data object returned by the wrapper (which automatically does an ismounted check) I believe you need to do one here as well.